src/lib.rs

```diff
@@ -394,35 +398,98 @@ impl Check {
                                 Some(version),
                             )?;
 
-                            Ok(run_check_release(
-                                &mut config,
-                                crate_name,
-                                current_crate,
-                                baseline_crate,
-                                self.release_type,
-                            )?)
+                            Ok((
+                                crate_name.clone(),
+                                Some(run_check_release(
+                                    &mut config,
+                                    crate_name,
+                                    current_crate,
+                                    baseline_crate,
+                                    self.release_type,
+                                )?),
+                            ))
                         }
                     })
                     .collect()
             }
         };
-        let success = all_outcomes
-            .into_iter()
-            .fold_ok(true, std::ops::BitAnd::bitand)?;
+        let crate_reports: BTreeMap<String, CrateReport> = {
+            let mut reports = BTreeMap::new();
+            for outcome in all_outcomes {
+                let (name, outcome) = outcome?;
+                if let Some(outcome) = outcome {
+                    reports.insert(name, outcome);
+                }
+            }
+            reports
+        };
 
-        Ok(Report { success })
+        Ok(Report { crate_reports })
     }
 }
 
+/// Report of semver check of one crate.
+#[non_exhaustive]
+#[derive(Debug)]
+pub struct CrateReport {
+    /// Bump between the current version and the baseline one.
+    detected_bump: ActualSemverUpdate,
+    /// Minimum bump required to respect semver.
+    /// For example, if the crate contains breaking changes, this is [`ReleaseType::Major`].
+    /// If no bump is required, this is [`Option::None`].
```

Comment:
Perhaps
```suggestion
    /// If no additional bump beyond the already-detected one is required, this is [`Option::None`].
```
or something similar?